MSAL/src/native_auth/network/parameters/MSALNativeAuthRequestContext.swift

```diff
@@ -29,10 +29,17 @@ class MSALNativeAuthRequestContext: MSIDRequestContext {
     private let _correlationId: UUID
     private let _telemetryRequestId: String
     private var _serverCorrelationId: UUID? // TODO: Setting the server correlation id here is wrong. Needs refactoring.
+    var disableFRT: Bool
```

Comment:
do we need it to be public property?

Reply:
_Property 'disableFRT' must be as accessible as its enclosing type because it matches a requirement in protocol 'MSIDRequestContext'_ 

This can change if there is an equivalent to MSALGlobalConfig in CommonCore (the other PR)

Reply:
This is no longer needed, removed.